src/EnergyPlus/DataHeatBalance.hh

```
@@ -1852,6 +1857,11 @@ namespace DataHeatBalance {
 		Real64 EMSFanPower; // Value EMS is directing to use for override of Fan power [W]
 		bool EMSUPSPowerOverrideOn; // EMS actuating UPS power if .TRUE.
 		Real64 EMSUPSPower; // Value EMS is directing to use for override of UPS power [W]
+		Real64 SupplyApproachTemp; // The difference of the IT inlet temperature from the AHU supply air temperature
+		int SupplyApproachTempSch; // The difference schedule of the IT inlet temperature from the AHU supply air temperature
+		Real64 ReturnApproachTemp; // The difference of the unit outlet temperature from the well mixed zone temperature
```

Same comment here as yesterday - "unit outlet" is ambiguous - it's not the outlet temp from the IT equipment, it's the return temp for the return node flowing into the CRAC.
This raises another question - can this model coexist with another HVAC system in the same zone?  If so, the IT equipment needs an input for the return air node name.  If not, then it should throw an error if the zone has more than one return node?

---

The change only applies for SingleDuct:VAV. We added check (warning) on if the HVAC system applies.